### ISSUE TITLE
Consumables bought in shop don't stack with a non-full stack until I relog or change PF

### ISSUE THREAD

**Issue by Nephima** [project-rubika/issues#963]
### Reproduction steps

```bash
1. Buy Basic Nano Recharger (stack of 50).
2. Use it so the stack drops to 49.
3. Buy new Basic Nano Recharger (stack of 50).
4. Try to merge the first 49 stack with new 50 stack.
5. Change area or relog and try again.
```

### Expected results

```bash
Newly bought consumables of the same QL should immediately stack with each other.
```

### Actual results

```bash
They don't merge with an item which has already been used until I change PF or relog.
```

### Bug prevalence

Every time.

### Description

```bash
n/a.
```

### Report link

https://pastebin.com/PgeDxADh

**Comment by RaggyLP** [comment]
Duplicate of #872.
